.travis.yml

```diff
@@ -17,12 +23,8 @@ install:
 - bundle install
 
 before_script:
-- echo 'before_script'
-- export AWS_REGION='us-east-1'
-- export TF_VAR_region=${AWS_REGION}
-- echo "using AWS_REGION=${AWS_REGION}"
-- export TF_WARN_OUTPUT_ERRORS=1
-- curl --silent --output terraform.zip https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip
+- export LATEST_TERRAFORM_VERSION=$(curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d '"' -f 4 | cut -c2-)
+- curl --silent --output terraform.zip https://releases.hashicorp.com/terraform/${LATEST_TERRAFORM_VERSION}/terraform_${LATEST_TERRAFORM_VERSION}_linux_amd64.zip
 - sha256sum terraform.zip  | grep "84ccfb8e13b5fce63051294f787885b76a1fedef6bdbecf51c5e586c9e20c9b7"
```

Comment:
I gather this line will start failing as we dynamically pull the latest terraform. If there's a way to also dynamically pull the checksum, I'm all for doing that. If that's too burdensome, I'd be okay to scrap the checksum altogether.

Reply:
that is good catch, I can remove this checksum, unless you have other thought.